plugins/kubernetes-backend/api-report.md

```diff
@@ -336,6 +337,9 @@ export type KubernetesObjectTypes =
   | 'statefulsets'
   | 'daemonsets';
 
+// @public
+export const kubernetesPlugin: () => BackendFeature;
```

Comment:
I think it's best to export this under `/alpha` like many other plugins have done tbh

Reply:
Sounds like a plan